BridgeSDK/BridgeAPI/SBBAuthManagerInternal.h

```diff
@@ -34,7 +34,15 @@
 
 extern SBBEnvironment gSBBDefaultEnvironment;
 
-@interface SBBAuthManager(internal)
+@protocol SBBAuthManagerInternalProtocol <SBBAuthManagerProtocol>
+
+- (BOOL)isAuthenticated;
+- (void)setSessionToken:(NSString *)sessionToken;
+- (void)clearSessionToken;
+
+@end
+
+@interface SBBAuthManager(internal)<SBBAuthManagerInternalProtocol>
```

Comment:
The unnamed "private" category is already being used in the .m file, and I didn't want to move everything from there to here. So, yes.